### ISSUE TITLE
Como realizar esta instalação?

### ISSUE THREAD

**Issue by adrianobr** [marcodotcastro/redmine_gamification_plugin_theme#1]
Este repositório complementa o de gamificattion?
Como realizar esta instalação?

**Comment by marcodotcastro** [comment]
Mais ou menos @adrianobr, ele é só um simples tema que estou criando para andar junto com o plugin mesmo que hoje exista uma pequena dependência. Essa pequena dependência é o icon de um menu no menu_admin.

**Comment by adrianobr** [comment]
sim, notei a falta do ícone quando clico em administração. Busquei em outros plugins para tentar intender o porque ele não adiciona o próprio CSS do plugin gamification, mas não encontrei a solução.
